### PR TITLE
+httpaf.0.6.2

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/httpaf-async/httpaf-async.0.6.2/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/httpaf-async/httpaf-async.0.6.2/opam]:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.5

File: packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.2/opam [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "faraday-lwt-unix"
+  "httpaf"
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.2/opam]:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.5

File: packages/httpaf/httpaf.0.6.2/opam [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/httpaf/httpaf.0.6.2/opam]:
```suggestion
  "dune" {>= "1.5"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore the `dune-project` file in the source repository says this package requires at least dune 1.5

File: packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.2/opam [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+bug-reports: "https://github.com/inhabitedtype/httpaf/issues"
+dev-repo: "git+https://github.com/inhabitedtype/httpaf.git"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "faraday-lwt-unix"
+  "httpaf"
```
Comment (kit-ty-kate) [inline packages/httpaf-lwt-unix/httpaf-lwt-unix.0.6.2/opam]:
```suggestion
  "httpaf" {= version}
```
Are you sure this package is compatible with all versions of `httpaf`?

File: packages/httpaf-async/httpaf-async.0.6.2/opam [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  "faraday-async"
+  "async"
+  "httpaf" {>= "0.5.0"}
```
Comment (kit-ty-kate) [inline packages/httpaf-async/httpaf-async.0.6.2/opam]:
```suggestion
  "httpaf" {>= version}
```
Is it really compatible with `httpaf.0.5.0`?

**Comment by mseri** [comment]
If you add `dune` lower bounds, we can merge. I don't have the rights to commit and merge the suggestion from @kit-ty-kate othwerwise I would have already done it

**Comment by kit-ty-kate** [comment]
superseded by #14993

**Pull request closed without merging**